autogpt/api/v1/endpoints/interactions.py

```diff
@@ -0,0 +1,52 @@
+from pathlib import Path
+
+from fastapi import APIRouter
+from pydantic.main import BaseModel
+
+from autogpt.agent import Agent
+from autogpt.api.v1.endpoints.agents import WorkspaceRequest
+from autogpt.config import AIConfig, Config
+from autogpt.memory.vector import get_memory
+from autogpt.prompts.prompt import DEFAULT_TRIGGERING_PROMPT
+from autogpt.workspace import Workspace
+from tests.integration.agent_factory import get_command_registry
+
+router = APIRouter()
+
+
+class InteractionRequest(BaseModel):
+    workspace: WorkspaceRequest
+    user_input: str
+
+
+@router.post("/agents/{agent_id}/interactions")
+async def create_interactions(agent_id: str, body: InteractionRequest):
+    config = Config()
```

Comment:
this acts as a link between the client and our specific implementation.
You can see we decide how to create this agent, by making arbitrary choices.
The alternative was to have the client do that, but again, we want an agnostic benchmark, so the client should be dumb and just ask for a task to do.